### PR TITLE
Feature/beaker secrets

### PR DESCRIPTION
**Pull request by jbrown-xentity**
Related to https://github.com/GSA/datagov-deploy/issues/2603
Add comments for ini file.
Followed new [secrets management process](https://github.com/GSA/datagov-deploy/wiki/Cloud.gov-Cheat-Sheet#secrets-management) to deploy the secrets; they are ready for the application start script to be deployed to be added to the service properly.

### CODE REVIEW

File: config/production.ini [hunk 1]
```diff
@@ -42,10 +42,6 @@ beaker.session.url=%(sqlalchemy.url)s
 beaker.session.timeout=3000
 beaker.session.lock_dir=/var/tmp/ckan/lock
 
-# `paster make-config` generates a unique value for this each time it generates
-# a config file.
-app_instance_uuid = {{ckan_instance_uuid}}
```
Comment (adborden) [inline config/production.ini:47]: Why remove this?

File: cfstart.sh [hunk 2]
```diff
@@ -37,6 +37,7 @@ REDIS_PORT=$(echo $VCAP_SERVICES | jq -r --arg SVC_REDIS $SVC_REDIS '.[][] | sel
 # We need the secret credentials for various application components (DB configuration, license keys, etc)
 DS_RO_PASSWORD=$(echo $VCAP_SERVICES | jq -r --arg SVC_SECRETS $SVC_SECRETS '.[][] | select(.name == $SVC_SECRETS) | .credentials.DS_RO_PASSWORD')
 export NEW_RELIC_LICENSE_KEY=$(echo $VCAP_SERVICES | jq -r --arg SVC_SECRETS $SVC_SECRETS '.[][] | select(.name == $SVC_SECRETS) | .credentials.NEW_RELIC_LICENSE_KEY')
+export CKAN___BEAKER_SESSION_SECRET=$(echo $VCAP_SERVICES | jq -r --arg SVC_SECRETS $SVC_SECRETS '.[][] | select(.name == $SVC_SECRETS) | .credentials.BEAKER__SESSION__SECRET')
```
[the remark this answers is not in the document]
Comment (jbrown-xentity) [reply cfstart.sh]: Good call/catch. Updated secret variable name to be same full path name.

File: config/production.ini (hunk 1 above)
Comment (jbrown-xentity) [reply config/production.ini:47]: This is no longer utilized in CKAN in any way. We had as part of the ticket to add it as a secret, but that becomes something we need to manage that we no longer need. See comment in thread starting [here](https://github.com/GSA/datagov-deploy/issues/2603#issuecomment-778002836)

Comment (adborden) [reply config/production.ini:47]: I still see the code in paster. Why not just set this to a guid? What if ckan does care about this value in the future or paster uses it for something that we didn't see?

Comment (adborden) [reply config/production.ini:47]: And the docs don't mention it as a secret so I would not treat it as such.

**Comment by jbrown-xentity** [comment]
Ready for another review @adborden

**Review by adborden — APPROVED** [review]

**Pull request merged**